### DISCUSSION TITLE
SpunOut in Taiko and possibly Catch

### DISCUSSION THREAD

**Discussion by TomyDoesThings** [ppy/osu#22794] [Ideas]
I find it strange that in Taiko, there are spinners/dendens/shakers and drumrolls, yet, SpunOut (SO) has never been implemented. And in Catch, there are technically spinners with a best route to follow in trying to catch the most spinner fruits.
In standard, SO helps in spinning for you if you're feeling burnt out in spinning. Why has this not been brought to other modes? Only mania wouldn't make sense for obvious reasons.

SpunOut should be considered in being added to Taiko and Catch.

**Comment by mk56-spn** [comment]
the reason it hasnt been done is most likely because nobody has bothered to do so, since this is the sort of thing core devs usually wouldn't be involved with mostly ( new mods are mostly by community members), im not aware of any opposition to bringing the mods to other rulesets
